src/plugins/validation/oas3/semantic-validators/responses.js

```diff
@@ -77,13 +77,20 @@ module.exports.validate = function({ resolvedSpec }, config) {
           }
         }
         // validate all success codes
-        if (!successCodes.length) {
+        if (!successCodes.length && !("101" in obj)) {
```

Comment:
I think I would have implemented this differently.  I would have changed `getResponseCodes` to return `101` in `successCodes`, since we want to treat it like one.  I'm curious why you chose this implementation instead.

Reply:
I actually did that first but it would have required changes to the check below that verifies that 204 is the only success code sans body.

Reply:
I'm good with this then.  We shouldn't over-think this.